src/Action.ts

```diff
@@ -70,11 +70,11 @@ export class ActionBase {
     this.isTerminal = action.isTerminal
     this.isEntryNode = action.isEntryNode
     this.repromptActionId = action.repromptActionId
-    this.requiredEntitiesFromPayload = action.requiredEntitiesFromPayload || []
-    this.requiredEntities = action.requiredEntities || []
-    this.negativeEntities = action.negativeEntities || []
-    this.requiredConditions = action.requiredConditions || []
-    this.negativeConditions = action.negativeConditions || []
+    this.requiredEntitiesFromPayload = action.requiredEntitiesFromPayload
```

Comment:
Why this change?  This could potentially leave these fields in "undefined" state.   Would that be valid?  I'm assuming they were set to empty arrays as that's not valid

Reply:
![truthy](https://user-images.githubusercontent.com/53840199/67226353-925cde00-f3e9-11e9-817e-03ad78874095.png)

Reply:
Hmm...  As the constructor doesn't take a partial, theoretically one shouldn't be able to pass in undefined values, but once it is transpiled to JS all bets are off.  @mattmazzola - what do you think?

Reply:
If you're confident that the constructor is never given a object that is not of type `ActionBase` object then it's likely good. I would try to find out why the `[]` were added in first place. Maybe the commit history has it

Reply:
Well, this change was just trying to chip away at the wall of warnings we get when we build this repo.  Per the linter message, it seems like the "|| []" here will _never_ have any effect.

I'm guessing the pre-allocation of empty arrays in the field definitions are also irrelevant since they're always overwritten in the ctor.